src/Qandidate/Common/Symfony/HttpKernel/EventListener/JsonRequestTransformerListener.php

```diff
@@ -20,6 +20,8 @@
  */
 class JsonRequestTransformerListener
 {
+    private $jsonContentTypes = ['json', 'application/json'];
```

Comment:
Just updated it.

Anyway after doing this I realized it isn't needed. Symfony automatically accepts json payloads. I was sending a malformed json which was causing the issue. 

Thanks anyway!